crates/bevy_input/src/keyboard.rs

```diff
@@ -407,3 +416,7 @@ pub enum KeyCode {
     /// The `Cut` key.
     Cut,
 }
+
+#[derive(Debug, Hash, Ord, PartialOrd, PartialEq, Eq, Clone, Copy)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+pub struct ScanCode(u32);
```

Comment:
This needs docs, and KeyCode and ScanCode should link to each other.

Reply:
How to make them link to each other?

Reply:
isn't linking gonna make this feature layout dependent?

Reply:
i could make example with text for when person clicks a button it's scan_code and key_code shown on screen

Reply:
I think Alice meant [this](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html).

Reply:
done, thanks for a link